### ISSUE TITLE
plot method should show skew-t fit and red vertical line showing threshold

### ISSUE THREAD

**Issue by lwaldron** [waldronlab/doppelgangR#11]
And should also pause between plots, e.g.:

``` r
library(curatedOvarianData)
library(doppelgangR)
data("GSE32063_eset")
data("GSE6008_eset")
esets <- list(GSE32063=GSE32063_eset, GSE6008=GSE6008_eset)
testdop <- doppelgangR(esets)
plot(testdop)
```

**Comment by lwaldron** [comment]
This is accomplished now, but now many of the PDFs look funky, e.g. see ovarian_dop.pdf that is produced by inst/ovarian.R.  Should create a better unit test for this.

**Comment by lwaldron** [comment]
Density line is fixed in 0.10.4.
